jujugui/static/gui/src/app/components/env-switcher/env-switcher.js

```diff
@@ -108,6 +109,18 @@ YUI.add('env-switcher', function() {
     },
 
     /**
+      Sets the state of the 'showEnvList' property to the inverse of what
+      it was when <Enter> or <Space> is clicked.
+
+      @method handleKeyToggle
+    */
+    handleKeyToggle: function(e) {
+      if (e.key === 'Enter' || e.key === ' ') {
```

Comment:
For some reason `keyCode` was returning `0` on all key presses but the `key` value was correct. Also seemed more explicit and easier to read. Is there a cross browser issue with using `key`?